app/components/data_table/cell_component.rb

```diff
@@ -0,0 +1,72 @@
+module DataTable
+  class CellComponent < GovukComponent::Base
+    def initialize(colname:, scope:, colspan:, sorting:, numeric:, text:, classes: []) # rubocop:disable Metrics/ParameterLists
+      @colname = colname
+      @text = text
+      @colspan = colspan
+      @numeric = numeric
+      @scope = scope
+      @sorting = sorting
+
+      super(classes: classes, html_attributes: {})
+    end
+
+    attr_reader :sorting, :colname, :colspan, :scope, :numeric, :text
+
+    def call
```

Comment:
Can this class produce anything other than a table header? Wondering if the name could change to `TableHeaderCellComponent`?

Reply:
Good point. It can't for now.